pages/glossary/accessible-description.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Accessible Description
+key: accessible-description
+unambiguous: true
+objective: true
+input_aspects:
+  - Accessibility tree
+  - CSS styling
+  - DOM tree
+---
+
+The _accessible description_ is the programmatically determined description of a user interface element that is [included in the accessibility tree](#included-in-the-accessibility-tree). The accessible description provides information that complements the [accessible name][].
```

Comment:
Never noticed that our def of "accessible name" is only for "UI element" 🤔 Feels a bit weird or restrictive (and maybe ambiguous since we usually use "instrument"). Again not related to this PR.